_posts/2018-04-16-the-hubris-of-man.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: The Hubris of Man
+date: 2018-04-16
+categories:
```

Comment:
I'm not exactly sure how the categories field parses, but at the very least, get rid of the commas at the end